jax/_src/numpy/polynomial.py

```diff
@@ -292,6 +295,25 @@ def polymul(a1, a2, *, trim_leading_zeros=False):
   val = convolve(a1, a2, mode='full')
   return val
 
+@_wraps(np.polydiv, lax_description=_LEADING_ZEROS_DOC)
+def polydiv(u, v, *, trim_leading_zeros=False):
+  _check_arraylike("polydiv", u, v)
+  u, v = _promote_dtypes_inexact(u, v)
+  w = u[0] + v[0] # calculate type
```

Comment:
This shouldn't be necessary, because the outputs of `_promote_dtypes_inexact` will all have the same dtype.